src/content/getting-started/developer-tools/cli.md

```diff
@@ -24,61 +24,16 @@ The easiest way to install the CLI is to open a Terminal and type:
 bash <( curl -sL https://particle.io/install-cli )
```

Comment:
@hugomontero did you intend on changing the install link? We can keep this vanity link as well. It's just a deploy of the website to change.

Reply:
we're gonna use this `bash <( curl -sL https://particle.io/install-cli )`  and change the link here: 👉 https://github.com/particle-iot/website/blob/master/static/_redirects#L215